src/install.coffee

```diff
@@ -355,6 +355,10 @@ class Install extends Command
             @installNode (error) -> next(error, packagePath)
         commands.push (packagePath, next) =>
           @installModule(options, pack, packagePath, next)
+        if installGlobally and packageName isnt pack.name
+          commands.push (newPack, next) => # remove package with old name
```

Comment:
I would make this comment a bit clearer; something like "package was renamed; delete old package folder"